### DISCUSSION TITLE
Give random mod a lower score multiplier

### DISCUSSION THREAD

**Discussion by DerHamm3r** [ppy/osu#31366] [Ideas]
Certain maps in the game are made with awkward, harder to hit angles. With a lucky seed you could use the random mod to potentially remove those awkwardness into something more comfortable, making a map easier to play. And since random mod applies to leaderboards it should in my opinion have a lower score multiplier to account for that.

**Comment by peppy** [comment]
Random mod should be isolated enough that it doesn't show up in normal leaderboards. As it would be disrespectful to mappers.
